ui/src/app/cluster-workflow-templates/cluster-workflow-template-list.scss

```diff
@@ -5,4 +5,229 @@
   &__row:hover {
     box-shadow: 1px 2px 3px rgba($argo-color-gray-9, .1), 0 0 0 1px rgba($argo-color-teal-5, .5);
   }
-}
\ No newline at end of file
+}
+
+.cluster-workflow-templates-list {
+  padding: 1em;
+
+  &__row-container {
+    a {
+      color: $argo-color-gray-6;
+    }
+
+    a:hover {
+      color: $argo-color-teal-5;
+    }
+  }
+
+  &__link-wrapper {
```

Comment:
I'm not sure why you copy+pasted all the CSS here -- the vast majority of it is not necessary and unused. For instance, the "statuses" below are entirely unused and also not related to Workflows (those are CD sync statuses).